tests/Unit/Visualization/Python/Test_PlotCce.py

```diff
@@ -0,0 +1,355 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import os
+import shutil
+import unittest
+
+import h5py
+import numpy as np
+from click.testing import CliRunner
+
+import spectre.IO.H5 as spectre_h5
+from spectre.Informer import unit_test_build_path
+from spectre.Visualization.PlotCce import plot_cce_command
+
+
+class TestPlotCce(unittest.TestCase):
+    def setUp(self):
+        self.work_dir = os.path.join(
+            unit_test_build_path(), "Visualization/PlotCce"
+        )
+        self.reduction_file_name_one_subfile = "TestCceReductions1.h5"
+        self.reduction_file_name_two_subfiles = "TestCceReductions2.h5"
+        plot_quantities = [
```

Comment:
(Optional) Is it possible to use the C++ routine that writes data in the simulation also to write data here? Would be nice to make sure the code keeps working.

Reply:
Not easily I don't think. This data is written from an iterable action that then calls either a threaded action or a local synchronous action to do the writing. The iterable action is pretty complex too